tests/bugfixes/github/test_pr_2072.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+import system_tests
+
+class NikonSigmaLens_APO_MACRO_180_F35_EX_DG_HSM(metaclass=system_tests.CaseMeta):
+    url = "https://github.com/Exiv2/exiv2/pull/992"
+    
+    filename = "$data_path/Sigma_APO_MACRO_180_F3.5_EX_DG_HSM.exv"
```

Comment:
There seems to be a copy/paste typo here. You are using a different file than the one added in this PR (also in the test name)